src/plugins/imagery/ImageryView.js

```diff
@@ -11,6 +11,8 @@ export default class ImageryView {
     }
 
     show(element) {
+        let isCompact = this.objectPath.find(object => object.type === 'time-strip');
```

Comment:
It really feels to me as though the timestrip view of imagery should be a completely separate view. It's really a completely different way of viewing images.

I think we should define a new view provider, with a `canView` function that basically includes the logic above. The `view` function of the new provider would return the much stripped-down view of images appropriate for the Time View. 

The `canView` function of the ImageryViewProvider would probably also need to be modified to make sure it does not apply when the view is embedded in a timestrip.

Reply:
You can use the `PreviewAction` to load the regular image view when an image is clicked on (see comments on this below), and use the Selection API for communicating which image has been clicked to the regular image view in its preview window.

Reply:
Ah!! I had this as a separate view before, but given the viewLarge functionality I brought the view into the Imagery Layout.
If PreviewAction works similarly to viewLarge then it definitely should be a separate view. I'll make the change.

Reply:
Done.